docs/pages/access-controls/idps/saml-grafana.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Authenticate to Grafana with Teleport
+description: Configure Grafana to use identities provided by Teleport.
+---
+
+Grafana is an open-source observability platform. Their enterprise version supports
+SAML authentication. This guide will help you configure Teleport as a SAML provider,
+and Grafana to accept the identities it provides.
+
+Note that Teleport can act as an identity provider to any SAML-compatible service,
+not just those running behind the Teleport App Service.
+
+## Prerequisites
+
+- An instance of Grafana Enterprise, with edit access to `grafana.ini`.
+  - A certificate solution to create TLS certificates/keys for the SAML connection.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Configure a Teleport role with access to SAML service provider objects
+
+First we need to ensure you are logged into Teleport as a user that has permissions
+to read and modify `saml_idp_service_provider` objects. The default `editor` role
+has access to this already, but in case you are using a more customized configuration,
+create a role called `sp-manager.yaml` with the following contents:
+
+```yaml
+kind: role
+metadata:
+  name: sp-manager
+spec:
+  allow:
+    rules:
+    - resources:
+      - saml_idp_service_provider
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+version: v6
```

Comment:
I find it easier to read if `version` is up with kind and metadata as opposed to after the spec.

Reply:
Moved.